docs/capabilities/architecture-quality.md

```diff
@@ -0,0 +1,60 @@
+# Architecture Quality
+
+Software engineering is a socio-technical activity. Software is built by teams of people, working together to produce a joint product. High performers in socio-technical architecture quality show faster issue resolution times. Sigrid measures Architecture Quality Model to quantify these aspects of socio-technical software architecture, covering both technical and social aspects and then compares the results against [SIG’s benchmark](../getting-started/approach.md). 
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/BejJ9sHeUOQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+When people talk about "legacy" or "monoliths", they often mean much more than just outdated technology. Rather, it refers to any pre-existing software solution that has become too fragile for changes to be timely, predictable, and reliable, usually due to poor architecture, or team knowledge loss. 
```

Comment:
Maybe the second sentence can follow as a continuation of the first? In this case, I guess we could write:
"What people usually mean is any pre-existing software solution that has become too fragile for changes to be timely, predictable, and reliable, usually due to poor architecture, or team knowledge loss.

Reply:
I'll also remove some comma's, since the writing style is noticeably more formal than the rest of this page.